RHIDP-14342: Admin guide documents catalog entity picker usage for Orchestrator workflows (#2547)
redhat-developer/red-hat-developers-documentation-rhdh: assemblies/extend_orchestrator-in-rhdh/assembly-enable-orchestrator-plugins-components.adoc

:context: enable-orchestrator-plugins-components

[role="_abstract"]

> You can enable the Orchestrator plugin components in your {product-short} instance and configure scaffolder templates with entity pickers and output links for running serverless workflows.

Reviewer: ```suggestion
To run serverless workflows in {product-short}, enable the Orchestrator plugin components and configure scaffolder templates with entity pickers and output links.
```
